test/services/pin/patch.test.js

```diff
@@ -237,45 +266,130 @@ describe('Pin - PATCH', () => {
       });
   });
 
-  it('returns 401 not allowing normal user to update other user\'s  pin', (done) => {
-    const newData = {
-      owner: normalUser._id, // eslint-disable-line no-underscore-dangle
-      $push: {
-        progresses: {
-          photos: ['New progress photo url'],
-          detail: 'New progress',
-        },
+  describe('allows all roles except normal user to path progresses of other`s pin', () => {
+    const allowedUpdatingUsers = [
+      {
+        role: DEPARTMENT_HEAD,
+        id: USER_DEPARTMENT_HEAD_ID,
+        email: 'department_head@youpin.city',
+        password: 'youpin_admin',
```

Comment:
Shall we change the passwords to be more related from now?
youpin_department_head